style.css

```diff
@@ -1,1990 +1,4132 @@
+    width: 100%;
+    height: 79%;
+    position: absolute;
+    top: 30px;
+    left: 0;
+    border-top-left-radius: 5rem;
+    z-index: -1;
+  }
+
+  .footer {
+    position: relative;
+    top: 50px;
+    width: 375px;
+    height: 850px;
+    left: -9px;
+    padding: 0;
+    display: flex;
+    flex-direction: column;
+    z-index: 10;
+    background: url(Images/Contactme.png);
+    background-position: 50% 50%;
+    background-size: cover;
+  }
+
+  .contact {
+    position: absolute;
+    width: 801px;
+    height: 56px;
+    left: 50px;
+    top: 80px;
+
+    /* Header / 2 */
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+
+    /* or 130% */
+    display: flex;
+    align-items: center;
+    text-align: center;
+
+    /* Neutrals/N0 */
+    color: #fff;
+  }
+
+  .info {
+    position: absolute;
+    width: 327px;
+    height: 102px;
+    left: 18px;
+    top: 150px;
+
+    /* Header / 2 */
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 300;
+    font-size: 20px;
+    line-height: 28px;
+
+    /* or 130% */
+    text-align: center;
+
+    /* Neutrals/N0 */
+    color: #fff;
+  }
+
+  input[type=text] {
+    position: relative;
+    width: 75%;
+    border-radius: 8px;
+    background: #fff;
+    border: 1px solid #cfd8dc;
+    margin: 10px 0;
+    height: 48px;
+    padding: 0 1rem;
+    top: 310px;
+    left: 20px;
+  }
+
+  input[type=email] {
+    position: relative;
+    width: 75%;
+    border-radius: 8px;
+    background: #fff;
+    border: 1px solid #cfd8dc;
+    margin: 10px 0;
+    height: 48px;
+    padding: 0 1rem;
+    top: 310px;
+    left: 20px;
+  }
+
+  textarea {
+    position: relative;
+    width: 75%;
+    border-radius: 8px;
+    background: #fff;
+    border: 1px solid #cfd8dc;
+    margin-top: 8px;
+    padding: 1rem;
+    resize: none;
+    top: 310px;
+    left: 20px;
+  }
+
+  button {
+    width: 140px;
+    height: 48px;
+    padding: 12px 16px;
+    cursor: pointer;
+    margin: 18px 0;
+    border-radius: 8px;
+    align-items: center;
+    display: flex;
+    flex-direction: row;
+    position: relative;
+    border-color: #fff;
+    top: 330px;
+    left: 20px;
+  }
+
+  button:focus {
+    background: #2230d2;
+    color: #fff;
+  }
+
+  button:hover {
+    background: #6070ff;
+    color: #fff;
+  }
+
+  .btn {
+    /* Button /Normal */
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+
+    /* identical to box height, or 141% */
+
+    display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+
+    /* Primary/B400 */
+    color: #6070ff;
+    background: #fff;
+  }
+
+  .left_block,
+  .left_block1,
+  .left_block2,
+  .left_block3,
+  .navmenu,
+  .menu,
+  .portfolio,
+  .about,
+  .Contact,
+  .handcursor {
+    display: none;
+  }
```

Comment:
- [ ] I observed that the details in the work cards disappear when switched to the mobile version. Please fix this as you should stick to the design in desktop and mobile versions.

## Your design
![d1](https://user-images.githubusercontent.com/94551484/186763208-46218368-7e67-4964-885e-69c65809755e.png)

## Figma template
![ccc](https://user-images.githubusercontent.com/94551484/186762880-2671f08b-8bdb-4e23-8a0a-56f8101fc6b9.png)
b15ff44.png)